stubs/redis/redis/cluster.pyi

```diff
@@ -180,6 +182,9 @@ class ClusterPipeline(RedisCluster[_StrType], Generic[_StrType]):
     reinitialize_steps: int
     encoder: Encoder
     commands_parser: Incomplete
+    scripts: Any
+    watching: bool
+    explicit_transaction: bool
```

Comment:
These were above the `reset()` method deliberately, I think -- it's because these attributes are only set in the `reset()` method, so it makes sense to keep them close to that method. Can you move them back?

Also, maybe `scripts` can be `set[Any]`? https://github.com/redis/redis-py/blob/b5ebada842c715e9ee74ea638a7e6d11afcddae1/redis/cluster.py#L1814